### PR TITLE
set launcher github repo description env var

### PR DESCRIPTION
**Pull request by JameelB**
Set the `LAUNCHER_BACKEND_GIT_REPOSITORY_DESCRIPTION` to display the correct Launcher url in the generated repo's description.

Updated the `launcher_template` to point to the latest version which includes the change for adding the `LAUNCHER_BACKEND_GIT_REPOSITORY_DESCRIPTION` environment variable.

image

### CODE REVIEW

File: evals/roles/launcher/defaults/main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 launcher_namespace: "{{ eval_launcher_namespace | default('launcher') }}"
 
-launcher_template: https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/cc0cabe7e2b541735cbea8f364d1fb98ef1f26ba/openshift/launcher-template.yaml
+launcher_template: https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml
```
Comment (pmccarthy) [inline evals/roles/launcher/defaults/main.yml]: Do we always want this to point to master? It means that we can no longer lock down the version of the template being used to a specific installer release.

Comment (pmccarthy) [inline evals/roles/launcher/defaults/main.yml]: Do we always want this to point to master? It means that we can no longer lock down the version of the template being used to a specific installer release.

**Comment by JameelB** [comment]
@pmccarthy updated, mind taking another look?

**Comment by aidenkeating** [comment]
@JameelB I'm seeing an issue during installation. Is this meant to be namespaced with `-n` or something?

```
TASK [launcher : Set Launcher GitHub repo description env var] ******************************************************************************************************************************************************************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {"changed": true, "cmd": "oc set env dc/launcher-backend LAUNCHER_BACKEND_GIT_REPOSITORY_DESCRIPTION=\"Generated by Red Hat Developer Launch (http://launcher-launcher.integreatly-demo2.opentry.me)\"", "delta": "0:00:00.141123", "end": "2018-10-23 14:50:51.855610", "msg": "non-zero return code", "rc": 1, "start": "2018-10-23 14:50:51.714487", "stderr": "Error from server (NotFound): deploymentconfigs.apps.openshift.io \"launcher-backend\" not found", "stderr_lines": ["Error from server (NotFound): deploymentconfigs.apps.openshift.io \"launcher-backend\" not found"], "stdout": "", "stdout_lines": []}
```

**Comment by JameelB** [comment]
@aidenkeating  good catch, will update that in a separate PR.

**Pull request merged**